packages/pn-personafisica-login/src/pages/loginError/LoginError.tsx

```diff
@@ -54,13 +56,14 @@ const LoginError = () => {
       case '1001':
         return t('loginError.code.error_1001');
       default:
-        return '';
+        return t('loginError.message');
     }
   };
 
+  const goToLogin = () => navigate(ROUTE_LOGIN);
   const title = t('loginError.title');
-  const message = (
-    <Trans i18nKey="loginError.message" ns="login">
+  const errorMessage = (
+    <Trans i18nKey={getErrorMessage()} ns="login">
```

Comment:
I'm not sure about this. The i18nKey wants the path to the label not the label itself.
For example 'loginError.message' and not t('loginError.message').

Furthermore {getErrorMessage()} is useless if we use the i18nKey.

We can remove the trans component because, in this case, is useless and use getErrorMessage() below, in the row where there is {errorMessage}